spec/class-model-snapshotted-views.spec.ts

```diff
@@ -72,27 +72,6 @@ describe("class model snapshotted views", () => {
     expect(instance.slug).toEqual("test");
   });
 
-  test("an observable instance emits a patch when the view value changes", () => {
```

Comment:
This test is already covered inside the `describe("shouldEmitPatchOnChange", ...)` tests.